File: spherevol.cpp
#include <iostream>
using namespace std;
    int main()
    {
    	float rad1;
    	float volsph;		
        cout<<" Input the radius of a sphere : ";
    	cin>>rad1;
    	volsph=(4*3.142*rad1*rad1*rad1)/3;
        cout<<" The volume of a sphere is : "<< volsph << endl;
        cout << endl;
        return 0;
    }

File: spherevol.java
double sphRad;
double sphVol;
System.out.println("Enter the radius of the sphere:");
sphRad = keyboard.nextDouble();
sphVol = ( 4.0 / 3.0 ) * Math.PI * Math.pow( sphRad, 3 );
//math.pi introduces pi to the equaton//
//math.pow() is a function in java that calculates the power of a number//
System.out.println("");
System.out.println("The volume is: " + sphVol);
